### PR TITLE
LIMS-6873

### PR DESCRIPTION
**Pull request by omarkasem**

### CODE REVIEW

**Comment by 0xIslamTaha** [comment]
@omarkasem 
- Solve conflicts
- add .vscode to .gitignore

File: ui_testing/testcases/basic_tests/test06_orders.py [hunk 1]
```diff
@@ -1886,3 +1886,26 @@ def test028_update_test_unit_with_delete_in_form(self):
         testunit_name = row_with_headers['Test Unit']
         self.base_selenium.LOGGER.info(" + Test unit : {}".format(testunit_name))
         self.assertIn(testunit_name, testunit_name)
+
+    def test01_archive_new_order_and_analysis(self):
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test06_orders.py]: - test01? The counter isn't right 
- Add test cases description as a docstr

File: ui_testing/testcases/basic_tests/test06_orders.py [hunk 2]
```diff
@@ -1886,3 +1886,26 @@ def test028_update_test_unit_with_delete_in_form(self):
...
+        all_orders = self.base_selenium.get_table_rows(element='orders:orders_table')
+        row_id = randint(0, len(all_orders) - 2)
+        main_order = self.base_selenium.get_row_cells_dict_related_to_header(row=all_orders[row_id])
+        sub_orders = self.orders_page.get_child_table_data(row_id)
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test06_orders.py]: use order API to get random order

**Comment by 0xIslamTaha** [comment]
@omarkasem 
can u check the `file changed`?
I can see that there is still .vscode files 
image

**Comment by omarkasem** [comment]
@0xIslamTaha I think it's removed now

**Comment by 0xIslamTaha** [comment]
@omarkasem 
no it is not. Please, check the Files changed and solve the conflict

**Comment by omarkasem** [comment]
> @omarkasem
> no it is not. Please, check the Files changed and solve the conflict

I see it's deleted from my end, what can i do to solve that issue ?

**Comment by omarkasem** [comment]
screenshot-github com-2020 02 28-18_01_08

File: ui_testing/testcases/base_test.py [hunk 3]
```diff
@@ -19,6 +39,30 @@ def setUp(self):
...
+        self.test_plan_api = TestPlanAPI()
+        self.test_unit_api = TestUnitAPI()
+        self.test_plan_api = TestPlanAPI()
+        self.orders_api = OrdersAPI()
+        self.analysis_api = AnalysisAPI()
+        self.single_analysis_page = SingleAnalysisPage()
+        self.contacts_api = ContactsAPI()
+        self.users_api = UsersAPI()
+        self.roles_api = RolesAPI()
+        self.general_utilities_api = GeneralUtilitiesAPI()
```
Comment (0xIslamTaha) [inline ui_testing/testcases/base_test.py]: Rebase from master we dont anymore initiate all those objects in the setup

File: ui_testing/testcases/base_test.py [hunk 4]
```diff
@@ -3,9 +3,29 @@
...
+from api_testing.apis.article_api import ArticleAPI
+from api_testing.apis.test_plan_api import TestPlanAPI
+from ui_testing.pages.header_page import Header
+from api_testing.apis.orders_api import OrdersAPI
+from ui_testing.pages.analysis_page import SingleAnalysisPage
+from api_testing.apis.contacts_api import ContactsAPI
+from api_testing.apis.users_api import UsersAPI
+from api_testing.apis.roles_api import RolesAPI
+from api_testing.apis.analysis_api import AnalysisAPI
+from api_testing.apis.general_utilities_api import GeneralUtilitiesAPI
```
Comment (0xIslamTaha) [inline ui_testing/testcases/base_test.py]: Rebase from master we dont anymore initiate all those objects in the setup

**Comment by omarkasem** [comment]
@0xIslamTaha I merged it by mistake, i reverted it back and also fixed the base test issue

**Pull request merged**